<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JobCrawlerGuiClass</class>
 <widget class="QMainWindow" name="JobCrawlerGuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>831</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Source Code Pro</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>JobCrawlerGui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="configureGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>601</width>
      <height>631</height>
     </rect>
    </property>
    <property name="title">
     <string>Configure</string>
    </property>
    <widget class="QGroupBox" name="urlGroupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>551</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>Url</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="urlPathLabel">
              <property name="text">
               <string>Path:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="urlPathVerticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="urlPageNumLabel">
            <property name="text">
             <string>PageNum:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPlainTextEdit" name="urlPathPlainTextEdit"/>
          </item>
          <item>
           <widget class="QSpinBox" name="urlPageNumSpinBox">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>552</width>
       <height>36</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="configureReloadPushButton">
        <property name="text">
         <string>Reload</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="configurePushButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="configureFileNameLabel">
          <property name="text">
           <string>FileName:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="configureFileNameLineEdit">
          <property name="text">
           <string>setting.json</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="filterSwitchGroupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>201</width>
       <height>148</height>
      </rect>
     </property>
     <property name="title">
      <string>FilterSwitch</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="filterSwitchJobContentCheckBox">
        <property name="text">
         <string>JobContent</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="filterSwitchJobTitleCheckBox">
        <property name="text">
         <string>JobTitle</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="filterSwitchToolEnableCheckBox">
        <property name="text">
         <string>Tool</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>948</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="JobCrawlerGui.qrc"/>
 </resources>
 <connections/>
</ui>
